Other-READMES/mac-homebrew-setup.md

```diff
@@ -30,7 +30,7 @@ $ brew install caskroom/cask/brew-cask
 ####Step 2: Install the Dependencies for Game of Life via Brew
 ```bash
 $ brew install maven
-$ brew install
+$ brew install jetty
```

Comment:
@jbankes - we should not need to install jetty ahead of time. Maven will do that when you execute mvn jetty:run